docs/swagger/spec.yml

```diff
@@ -3156,6 +3156,9 @@ definitions:
         description: output export path
       render:
         $ref: '#/definitions/Render'
+      dropboxCredential:
```

Comment:
Should we store the dropbox credential with the export? Or should we do something like rely on the dropbox credential for the user that is doing the export?

I kind of lean towards the second because if there's a situation where we have to delete a user's dropbox or it changes there's only one location that it is stored.